spec/amber/scripts/environment_loader_spec.cr

```diff
@@ -10,15 +10,16 @@ describe Amber::Server do
       @log = ::Logger.new(STDOUT).tap{|l| l.level = ::Logger::INFO}
       @color = true
       @redis_url = "\#{ENV[%(REDIS_URL)]? || %(redis://localhost:6379)}"
+      @database_url = "mysql://root@localhost:3306/amber_test_app_test"
```

Comment:
I seriously think that `database_url` should be in secrets since a database isn't required to run amber, or someone might want to have multiple database connections. Secrets is perfect for this since you could do `database_url`, `logging_database_url`, `kelly_blue_book_database_url`